src/gui/tracks/InstrumentTrackView.cpp

```diff
@@ -393,4 +398,27 @@ QMenu * InstrumentTrackView::createMixerMenu(QString title, QString newMixerLabe
 }
 
 
+QPixmap InstrumentTrackView::determinePixmap(InstrumentTrack* instrumentTrack)
```

Comment:
It feels a bit odd that this isn't static, but still takes the instrument track as an argument rather than using the `InstrumentTrackView`'s `model()`.